cmd/contrib/main.go

```diff
@@ -518,7 +518,10 @@ func {{ .PlatformNameUpperCamelCase }}CLI() schema.Executable {
 		Name:      "{{ .PlatformName }} CLI", // TODO: Check if this is correct
 		Runs:      []string{"{{ .Executable }}"},
 		DocsURL:   sdk.URL("https://{{ .Name }}.com/docs/cli"), // TODO: Replace with actual URL
-		NeedsAuth: needsauth.NotForHelpOrVersion(),
+		NeedsAuth: needsauth.For(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
+		),
```

Comment:
Any CLIs in our existing plugin catalog I'm missing that should also be using this new empty-args helper that opts out of authentication when no args are specified?